doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -93,7 +93,18 @@ Matter
 * For devices that use Matter over Thread, the default cryptography backend is now Arm PSA Crypto API instead of Mbed TLS, which was used in earlier versions.
   You can still build all examples with deprecated Mbed TLS support by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``, but you must build the Thread libraries from sources.
   To :ref:`inherit Thread certification <ug_matter_device_certification_reqs_dependent>` from Nordic Semiconductor, you must use the PSA Crypto API backend.
-* Updated the page about :ref:`ug_matter_device_low_power_configuration` with the information about Intermittently Connected Devices (ICD) configuration.
+
+* Updated:
+
+  * The :ref:`ug_matter_device_low_power_configuration` page with the information about Intermittently Connected Devices (ICD) configuration.
+  * The Factory Data Guide and added a new section explaining several important Kconfig options.
+
+* Changed the factory data default usage:
+
+  * The SPAKE2+ verifier is now generated by default with each build, and it will change if any of the related Kconfig options are modified.
```

Comment:
I think we need to remove now the entry in known-issue about this issue.

Reply:
Wouldn't this just mean that we don't update the issue as affecting any upcoming versions? It still affects the v2.5-based releases, so should remain in the known issues.

Reply:
Ok, that makes sense!